### ISSUE TITLE
Automatic PostProcessing not working w/ Deluge (possibly effects more clients)

### ISSUE THREAD

**Issue by xbullet** [SiCKRAGETV/sickrage-issues#1004]
BRANCH: (master) / COMMIT: (cc4896e96a27dda5cc9ed198209aa514bf68a4eb) 
Python Version: 2.7.8 (default, Sep 24 2014, 18:26:21) [GCC 4.9.1 20140903 (prerelease)]
OS: Arch Linux 3.17 Kernel

Log file:
There are no associated log entries since it never tries to process the files...

Issue:
When automatic/manually started downloads finish, they are not being post-processed automatically and need to be post-processed by using the manual post-processing button. This has been a longstanding issue for a number of months now with all versions.

Note, I've also tried the dev build w/ the same issue (was previously using the dev for a number of months)

Same as per following thread.
https://sickrage.tv/forums/forum/main-category/main-forum/9991-postprocessing-not-happening

**Comment by fernandog** [comment]
Follow guidelines:
https://github.com/SiCKRAGETV/sickrage-issues

**Comment by abeloin** [comment]
@xbullet As @fernandog stated we need more info, also these specific info:
* logs
* Config.ini:
  * autopostprocesser_frequency
  * process_automatically
  * tv_download_dir
* Deluge
  * path when completed

Thanks

**Comment by xbullet** [comment]
I will provide some further info soon guys, cheers

**Comment by fernandog** [comment]
@xbullet provide logs or will close this

**Comment by abeloin** [comment]
@fernandog Closing the OP will reopen when come back.

**Comment by xbullet** [comment]
FYI, I have found the issue to be caused by having too many items in postprocessing folder (100-200 already processed torrents), perhaps the background postprocessor threads are dying?

I've no time to sort logs and whatnot at the moment, I'm at work and the issue is no longer occurring for me since I've cleared the postprocessor folder, just a heads up for you guys.

What is best practice? Should I be moving post-processed files into a new folder to seed?
